client/card-readers/settings/sections/address-details.js

```diff
@@ -0,0 +1,83 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { TextControl, SelectControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useAccountBusinessSupportAddress } from '../../../data';
+
+const ReadersSettingsFields = () => {
+	const [
+		accountBusinessSupportAddress,
+		accountBusinessSupportAddressCountry,
+		accountBusinessSupportAddressLine1,
+		accountBusinessSupportAddressLine2,
+		accountBusinessSupportAddressCity,
+		accountBusinessSupportAddressPostalCode,
+		setAccountBusinessSupportAddress,
+	] = useAccountBusinessSupportAddress();
+
+	const handleAddresPropertyChange = ( property, value ) => {
+		setAccountBusinessSupportAddress( {
+			...accountBusinessSupportAddress,
+			[ property ]: value,
+		} );
+	};
+
+	const countriesOptions = Object.entries(
```

Comment:
The names of a few countries are not displaying correctly:

<img width="620" alt="image" src="https://user-images.githubusercontent.com/1288616/143569878-c9a36a70-309d-4657-a2e1-cd5d9b89fcd4.png">

Reply:
Fixed.